lighthouse-core/report/html/report-styles.css

```diff
@@ -1010,16 +1011,21 @@
 
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
-  color: var(--report-text-color-secondary);
+  color: var(--report-text-color);  
   margin-left: auto;
   margin-right: auto;
-  max-width: calc(var(--report-width) - var(--category-padding) * 2);
-  background-color: var(--color-amber-50);
+  max-width: calc(var(--report-width) - var(--category-padding) * 2);  
+  background-color: rgba(var(--color-orange-rgb), .1);
```

Comment:
introduce a new variable `--toplevel-warning-background-color`

Instead of using alpha, I think we want it set to orange 100 (#FFCCBC) in light mode and #544B40 for dark mode (I don't see a clean fit from the material ui colors, oh well)

initial value: `var(--color-orange-100)`
dark mode: `#544B40` (unless you can find a good material ui color)

Reply:
Corrected this, although I set the light mode background color to orange-50 instead of orange-100 as I felt this was a closer match to the original color